### PR TITLE
docs: JR-575: Updates Service Registry 'Using Quarkus' with PM+QE fee…

### PR DESCRIPTION
**Pull request by bredamc**
…dback

### CODE REVIEW

File: docs/registry/quarkus-registry/README.adoc [hunk 1]
```diff
@@ -158,24 +158,24 @@ endif::[]
...
-* You have the Core Registry API endpoint for the {registry} instance. To get this information, select your {registry} instance in the {service-url-registry}[{product-registry} web console^], select the options icon (three vertical dots) and click *Connection*. Copy the *Core Registry API* endpoint supported by the Apicurio serializer/deserializer (SerDes) used in this example.
-* You have the SASL/OAUTHBEARER token endpoint used by the {registry} and Kafka instances. To get this information, select your {registry} instance in the {service-url-registry}[{product-registry} web console^], select the options icon (three vertical dots) and click *Connection*. Copy the *Token endpoint URL* value.
+* You have a service account with write access to Kafka and {registry} instances and have stored your credentials securely (see {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^] and {base-url}{getting-started-url-registry}[Getting started with {product-long-registry}^]). Copy the *Client ID* and *Client secret* values.
```
Comment (smccarthy-ie) [inline docs/registry/quarkus-registry/README.adoc:168]: John recently updated these prereqs along with the Kafka Quarkus prereqs. You might want to also tweak the Kafka Quarkus prereqs for consistency (e.g., to include `Copy the *Client ID* and *Client secret* values` and `Copy the *Bootstrap server* value`?

File: docs/registry/quarkus-registry/README.adoc [hunk 2]
```diff
@@ -207,20 +207,15 @@ The Quarkus application in this quick start uses a Kafka topic called `quotes` t
...
-. Click *Create topic* and follow the guided steps to define the topic details.
-+
---
-You must specify the following topic properties:
-
-* *Topic name*: For this quick start, enter `quotes` as the topic name.
-* *Partitions*: Set the number of partitions for the topic. For this quick start, set the value to `1`.
-* *Message retention*: Set the message retention time and size. For this quick start, set the retention time to `A week` and the retention size to `Unlimited`.
-* *Replicas*: For this release of {product-kafka}, the replica values are preconfigured. The number of partition replicas for the topic is set to `3` and the minimum number of follower replicas that must be in sync with a partition leader is set to `2`. For a trial Kafka instance, the number of replicas and the minimum in-sync replica factor are both set to `1`.
+. In the {product-kafka} {service-url-kafka}[web console^], click *Kafka Instances* and then click the name of the Kafka instance that you want to add a topic to.
```
Comment (smccarthy-ie) [inline docs/registry/quarkus-registry/README.adoc:210]: Similarly, you might want to make similar tweaks the Kafka Quarkus QS for consistency? AFAIR, John also updated both of these recently.

**Review by smccarthy-ie — APPROVED** [review]
LGTM!

(with RHOSAK+Quarkus PR to follow soon to keep both guides in sync)

**Comment by bredamc** [comment]
@smccarthy-ie Made the changes in the RHOSAK+Quarkus doc now, while the changes are still fresh in my mind! Please review again to make sure I caught everything.

@AedinC Will you please review the changes in the RHOSAK+Quarkus doc too, to make sure you're happy with them?

**Review by AedinC — APPROVED** [review]
LGTM- thanks Breda.

File: docs/kafka/quarkus-kafka/README.adoc [hunk 3]
```diff
@@ -153,33 +153,28 @@ endif::[]
...
 
 .Prerequisites
-* You have the bootstrap server endpoint and the SASL/OAUTHBEARER token endpoint for the Kafka instance. To get the server endpoint and the SASL/OAUTHBEARER token endpoint, select your Kafka instance in the {product-kafka} {service-url-kafka}[web console^], select the options icon (three vertical dots), and click *Connection*.
-* You have the generated credentials for your service account. To reset the credentials, use the {service-accounts-url}[Service Accounts^] page in the *Application Services* section of the Red Hat Hybrid Cloud Console.
-* You've set the permissions for your service account to access the Kafka instance resources. To verify the current permissions, select your Kafka instance in the {product-kafka} web console and use the *Access* page to find your service account permission settings.
+* You have the bootstrap server endpoint and the SASL/OAUTHBEARER token endpoint for the Kafka instance. To get this information, find your Kafka instance in the {product-kafka} {service-url-kafka}[web console^], click the options icon (three vertical dots), and click *Connection*. Copy the *Bootstrap server* and *Token endpoint URL* values.
```
[the remark this answers is not in the document]
Comment (bredamc) [reply docs/kafka/quarkus-kafka/README.adoc:161]: Thanks, @AedinC -- I think it's just regular _IBM Style_ advice so it's not explicitly mentioned.
The verb to use depends on the UI element: we use _click_ for icons, menu items, navigation trees, and links (among others). 

Good idea to create an issue to review the rest of our docs for consistency :)

**Pull request merged**